modules/core/src/test/java/org/apache/ignite/internal/metric/JmxMetricExporterSpiTest.java

```diff
@@ -131,17 +134,70 @@ public void testFilterAndExport() throws Exception {
 
         assertThrowsWithCause(new RunnableX() {
             @Override public void runx() throws Exception {
-                metricSet(ignite.name(), "filtered", "metric");
+                metricRegistry(ignite.name(), "filtered", "metric");
             }
         }, IgniteException.class);
 
-        DynamicMBean bean1 = metricSet(ignite.name(), "other", "prefix");
+        DynamicMBean bean1 = metricRegistry(ignite.name(), "other", "prefix");
 
         assertEquals(42L, bean1.getAttribute("test"));
         assertEquals(43L, bean1.getAttribute("test2"));
 
-        DynamicMBean bean2 = metricSet(ignite.name(), "other", "prefix2");
+        DynamicMBean bean2 = metricRegistry(ignite.name(), "other", "prefix2");
 
         assertEquals(44L, bean2.getAttribute("test3"));
     }
+
+    /** */
+    @Test
+    public void testHistogramSearchByName() throws Exception {
```

Comment:
I think we should add the case where histogram intervals change.

Reply:
For now, we don't support interval changes.
So, I don't think we should test it.

Seems, we should re-register MBean if some metric change its properties.